docs/docs/developers/versions-updating.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Versions and Updating
+---
+
+
+## Versions
+Aztec tools (sandbox, cli, nargo), dependencies (aztec-nr), and sample contracts are constantly being improved.
+When developing and referring to example .nr files/snippets, it is helpful to verify the versions of different components (below), and if required keep them in lock-step by [updating](#updating).
+
+### Checking tool versions
+To check your version of Aztec tools, you can use `aztec-cli -V`
+
+:::note
+The `aztec-nargo` versions follow `nargo` versions, which is different to the Aztec tool versions.
+:::note
+
+The latest version of the Aztec tooling is currently `#include_aztec_version` , updating roughly every week.
+
+### Dependency versions
+Dependency versions in a contract's `Nargo.toml` file correspond to the `aztec-packages` repository tag `aztec-packages` (filter tags by `aztec`...)
+
+If you get an error like: `Cannot read file ~/nargo/github.com/AztecProtocol/aztec-packages/...`
+Check the `git=` github url, tag, and directory.
+
+:::note
+The folder structure changed at **0.24.0** from `yarn-project/aztec-nr` to `noir-projects/aztec-nr`.
```

Comment:
This is mentioned in the migration notes page. Consider linking there to reduce redundant information. Any developers coming after v0.24 don't need to see this